swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
 class Item:
-    pass
+    def __init__(self, category="", condition=0, age=0):
+        self.category = category
+        self.age = age
+        if condition != 0:
+            self.condition = condition
```

Comment:
We do not need to do this check. In fact, even if condition is the default value of 0, we want to set `self.condition` to `condition`. This is the code in the constructor the sets the value of the instance variable `condition`

```suggestion
        self.condition = condition
```